src/utils/validateCalculator.js

```diff
@@ -0,0 +1,33 @@
+import { ERROR_MESSAGE } from '../constants/errorMessage.js';
+import { CALCULATOR_CONFIGURE } from '../constants/calculatorConfigure.js';
```

Comment:
```suggestion
import { ERROR_MESSAGE, CALCULATOR_CONFIGURE } from '../constants';
```
이렇게 표현할 수 있도록 index.js 를 활용하면 어떨까요?